check_https.py

```diff
@@ -105,7 +105,7 @@ def fetch_through_redirects(url):
             resp.close()
         # Check for sneaky <meta> redirects.
         for meta in META_XPATH(tree):
-            m = re.match("0;\s*url=['\"](.+?)['\"]", meta.get("content"))
+            m = re.match(r"0;\s*url=['\"](.+?)['\"]", meta.get("content"))
```

Comment:
This isn't actually equivalent right? `len(r"\"") == 2``

Reply:
I think it still does the right thing?

```
alexgaynor@penguin ~> python
Python 2.7.13 (default, Sep 26 2018, 18:42:22) 
[GCC 6.3.0 20170516] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> re.search(r"['\"]", "abc\"")
<_sre.SRE_Match object at 0x7afe8f12e510>
>>> re.search(r"['\"]", "abc\"").group()
'"'
>>> 
```